src/App.js

```diff
@@ -1,62 +1,50 @@
-import React, { useEffect } from "react";
```

Comment:
It looks like your code formatter changed a lot of things. I know we didn't discuss how our codebase should be formatted in the beginning, but large changes like these can often cause merge conflicts and make it difficult to see what was changed.

Reply:
agree. take one more look at everything

Reply:
If you skim through, you can see I just added one line. No need to look through everything, but yes thank you Ace, I'll be sure to turn off the formatting next time.